forte/processors/base/data_selector_for_da.py

```diff
@@ -31,19 +31,19 @@
 
 
 __all__ = [
-    "BaseElasticSearchDataSelector",
-    "RandomDataSelector",
-    "QueryDataSelector",
+    "BaseElasticSearchDataSelectorForDA",
```

Comment:
I suppose you added 'ForDA' to avoid confusing with `Selector`? I think you have already named them as `DataSelector` it is actually OK now.

Reply:
Do you mean the previous names are ok and we can change them back?

Reply:
Yeah. I think the base `Selector` name will be confusing. These concrete classes look ok.